src/utils.ts

```diff
@@ -209,6 +210,20 @@ export class Utils {
         if (ruleIf === undefined) return true;
         let evalStr = ruleIf;
 
+        const flagsToBinary = (flags: string): number => {
+            let binary = 0;
+            if (flags.includes("i")) {
+                binary |= RE2JS.CASE_INSENSITIVE;
+            }
+            if (flags.includes("g")) {
```

Comment:
```suggestion
            if (flags.includes("s")) {
```
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/dotAll